packages/client/e2e/utils.ts

```diff
@@ -15,6 +15,11 @@ export enum TestTimeout {
   EXTENDED = 90000,
 }
 
+export enum RfqTimeout {
```

Comment:
we have Credit, with RFQs .. this enum name is potentially confusing
also, no need to move these here .. they are not general timeouts .. I specifically moved/kept them in the spot tile spec so they are located in the relevant scope